server/scrape_negative

```diff
@@ -12,17 +12,23 @@ if __name__ == "__main__":
     parser = argparse.ArgumentParser(prog="scraper")
     parser.add_argument("-n", "--narticles", type=int, action="store", default=0)
     args = parser.parse_args()
+    if args.narticles <= 0:
+        print("Sorry, values <= 0 are not valid numbers for articles to be scraped\nExiting")
+        sys.exit(1)
+    if len(NEGATIVE_QUERIES) == 0:
+        print("No negative queries have been supplied in .env\nExiting")
+        sys.exit(1)
     from database.mock_database import DatabaseNegative
 
     for query in NEGATIVE_QUERIES:
-        print("scraping for query [{}]".format(query))
```

Comment:
this we can also move to the google scraper to avoid repeating the same line in `scrape_positive`

Reply:
fixed in [5262b14](https://github.com/jannis-baum/weBlock/pull/17/commits/5262b1474616134bc9ee959b4eb9c83d3159525f)